### ISSUE TITLE
Refactoring for different device sizes

### ISSUE THREAD

**Issue by aakashs** [nzff/cardboard-swift#1]
Hi!

Thank you for this, it looks really useful.

It currently seems like viewport sizes for each eye are hardcoded for the iPhone 6+. Do you know which class/variables to modify to make this more dynamic across devices?

Thank you!

**Comment by nzff** [comment]
Hehe yeah good catch. ScreenParams.swift has what you're looking for I think. There's even a TODO note about what's missing.

What device are you testing on? 

I only have an iPhone6+, mind showing me a screenshot of what things look like as well

**Comment by aakashs** [comment]
I'm testing on an iPhone 6 and it currently looks like this:
![img_9362](https://cloud.githubusercontent.com/assets/2073966/13110896/a77619de-d550-11e5-9aba-b1c7afee319d.PNG)

Looking through the ScreenParams file, it seems like the pixelsPerInch() method is the one that needs modification?

**Comment by aakashs** [comment]
Hi, to follow up on this - 

I don't think ScreenParams actually contains the issue, because the size of the viewports actually seems to be correct across devices. The only thing that needs to be changed is the space in the middle between the viewports, which I can't seem to figure out how to adjust. Thoughts?

**Comment by nzff** [comment]
Yep you're right (there's a bug in ScreenParams still, but I'll get to that one later :dash: )

I just found and fixed the bug though, so you should be good to go. Since I don't have an actual device to test myself, let me know and then I'll close the issue.

**Comment by aakashs** [comment]
Yes, it's now sized correctly, thanks!

However (and this may be a separate issue), when you view your iphone 6+ through the google cardboard, do you notice your field of view is essentially a square with visible black borders?

The cardboard demo built in unity seems to have a wider field of view, and I think it might have to do with the viewport sizing/curvature/position.

Attached are screenshots of the swift version, followed by the unity version.

![img_9372](https://cloud.githubusercontent.com/assets/2073966/13204386/f189be46-d89c-11e5-859c-0b7215de5f53.PNG)
![img_9373](https://cloud.githubusercontent.com/assets/2073966/13204387/f5a47e08-d89c-11e5-9120-96c1a27a1f48.PNG)

Let me know what you think!
